easybuild/easyconfigs/c/CondaCreate/requirements.txt

```diff
@@ -0,0 +1 @@
+perl-app-cpanminus
```

Comment:
@jerowe not sure what you mean... how `conda_packages` is interpreted is up to the `EB_CondaCreate` easyblock?